dir.props

```diff
@@ -259,6 +259,7 @@
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(IntermediateOutputRootPath)$(MSBuildProjectName)/$(TargetOutputRelPath)</IntermediateOutputPath>
 
     <RuntimePath Condition="'$(RuntimePath)' == ''">$(BinDir)runtime/$(BuildConfiguration)/</RuntimePath>
+    <ShimsTargetRuntimePath>$(BinDir)shimsTargetRuntime/netcoreapp2.0/</ShimsTargetRuntimePath>
```

Comment:
Don't include netcoreapp2.0 as part of the path.  Make it just the root then append the targetGroup that's appropriate.

Reply:
But targetGroup will be either netcoreapp2.0 or netstandard, so then you mean that we should get the references as  a wildcard including the subdirs in the root? i.e: Root\**\*.dll?

Reply:
Let me be more explicit:
```
<PropertyGroup>
  <ShimsTargetRuntimeRoot>$(BinDir)shimsTargetRuntime/</ShimsTargetRuntimeRoot>
</PropertyGroup>
 <ItemGroup>
  <BinPlaceConfiguration Include="netcoreapp2.0-Windows_NT">
    <RuntimePath>$(ShimsTargetRuntimeRoot)netcoreapp2.0/</RuntimePath>
  </BinPlaceConfiguration>
 </ItemGroup>
```

Then in the shims projects:
```
  <_RuntimePath Condition="'$(TargetGroup)' == 'netcoreapp2.0'">$(ShimsTargetRuntimeRoot)netcoreapp2.0/</_RuntimePath>
```